api/turing/models/ensembler.go

```diff
@@ -27,7 +28,7 @@ type EnsemblerType string
 const (
 	EnsemblerStandardType EnsemblerType = "standard"
 	EnsemblerDockerType   EnsemblerType = "docker"
-	EnsemblerTypePyFunc   EnsemblerType = "pyfunc"
+	EnsemblerPyFuncType   EnsemblerType = "pyfunc"
```

Comment:
Renamed the constants to create more consistency in the naming, though technically, the naming conventions originally didn't quite serve the same purpose... `EnsemblerTypePyFunc` was used only for batch ensemblers, which were a separate thing from the other two types which are only used for ensembler configs for routers.